src/components/Datatable/Table/Table.styles.ts

```diff
@@ -178,6 +183,8 @@ export const StyledTable = styled.table`
     }
 
     .ds-table-checkbox-label {
```

Comment:
Do we support label for select checkbox? :) I can't find it in Storybook. If yes it would worth to add the story

Reply:
Can we also remove radio styles? They are still used or you want to keep them?

Reply:
No, we don't support the label, but label tag is what actually provides the clicking function on custom-styled checkbox:) I'll remove the radio styles, I forgot them

Reply:
Got it, thanks for explanation